packages/server/api/src/app/database/seeds/openops-delete-old-opportunities-table.ts

```diff
@@ -65,4 +64,6 @@ export const deleteOldOpportunitiesTable = async (): Promise<void> => {
       );
     }
   }
+
+  await setOpportunitiesTableDeleted();
```

Comment:
why is this out if the try catch?

Reply:
hmm, where would you put it? once in the try and once in the catch after the first if?